buttons-examples/iOS/ios-button.md

```diff
@@ -24,444 +26,332 @@ There are four types of buttons:
-* [Themes class description](https://material.io/develop/ios/components/theming/)  <!-- This is slated to be deprected, though the examples from https://material.io/develop/ios/components/buttons/api-docs/Classes/MDCButton.html appear to use this class -->
-* [MDCContainerScheme class description](https://github.com/material-components/material-components-ios/tree/stable/components/schemes/Container)
+&nbsp;                                          | Attribute         | Related method(s)                                         | Default value
+----------------------------------------------- | ----------------- | --------------------------------------------------------- | -------------
+**Icon**                                        | `imageView`        | `setImage:forState:`<br/>`imageForState:` | `nil`
+**Color**                                       | `imageView.tintColor` | `setImageViewTintColor:forState:`<br/>`imageViewTintColorForState:` | `nil`
 
-!["Outlined button example in Android with purple text surrounded by a gray outline"](assets/outlined-button.svg)
+## Outlined button
 
-  <!--<div class="material-code-render" markdown="1">-->
-**Swift**
-```swift
-let button = MDCButton()
-button.applyTextTheme(withScheme: MyMaterialTheme)
-```
-**Objective-C**
-```objc
-MDCButton *button = [[MDCButton alloc] init];
-[self.button applyTextThemeWithScheme:self.MyMaterialTheme];
-```
-  <!--</div>-->
+<img src="assets/outlined.gif" alt="An animation showing a Material Design outlined button." width="115">
 
-<!--
-<b>Adding an icon to a contained button</b>
+[Outlined buttons](https://material.io/components/buttons/#outlined-button) are medium-emphasis buttons. They contain actions that are important, but aren’t the primary action in an app.
 
-The following example shows a contained button with an icon.
+### Outlined button example
 
-!["iOS contained button with purple text 'Contained' and '+' icon over a white background."](assets/contained-button-icon.svg)
--->
+To achieve an outlined button use the outlined button theming method on the MDCButton theming extension. To access the theming extension see the [Theming section](#theming). 
 
-  <!--<div class="material-code-render" markdown="1">-->
-<!--
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
 ```swift
-
+button.applyOutlinedTheme(withScheme: containerScheme)
 ```
 
+#### Objective-C
 
 ```objc
-
+[self.button applyOutlinedThemeWithScheme:self.containerScheme];
 ```
--->
-  <!--</div>-->
-
-</details>
-
+<!--</div>-->
 
-### Anatomy and key properties
+### Anatomy and Key properties
 
-An outline button has text, a container, and an optional icon.
+An outlined button has a text label, a container, and an optional icon.
 
-![Outlined button anatomy diagram](assets/outlined_button_anatomy.png)
+![Outlined button anatomy diagram](docs/assets/outlined-button-diagram.png)
 
-1. Text label
-1. Container
-1. Icon
+A. Text label<br>
```

Comment:
There should be a version of the anatomy diagram that uses numbers instead of letters.

Reply:
Using those now, thanks!